birdie/src-tauri/src/system/terminal.rs

```diff
@@ -0,0 +1,22 @@
+use std::process::Command;
+use std::sync::Arc;
+
+use axum::extract::State;
+use tracing::error;
+
+use crate::state::AppState;
+
+#[cfg(any(target_os = "windows", target_os = "linux"))]
```

Comment:
powershell isn't a thing in linux so i'd recommend doing only windows here, then implementing a second function called `open_terminal_to_path` with the same signature that simply logs an error for not windows